examples/arm/setup.sh

```diff
@@ -56,7 +56,7 @@ fi
 
 # vela
 vela_repo_url="https://gitlab.arm.com/artificial-intelligence/ethos-u/ethos-u-vela"
-vela_rev="8cac2b9a7204b57125a8718049519b091a98846c"
+vela_rev="d37febc1715edf0d236c2ff555739a8a9aadcf9a"
```

Comment:
So IIUC this is a BC breaking change for Vela i.e. this new version can't run both v0.80 and v1.0, right since they hardcode the schema?

Reply:
Thats my understanding, old version 0.80.1 only and new version TOSA 1.0 only.